todo_list_class.rb

```diff
@@ -0,0 +1,63 @@
+require_relative 'todo_item_class.rb'
+
+class ToDoList
+	def initialize(name = "TODO")
+		@name = name
+		@list = Hash.new #maps name of item to item object
+	end
+
+	def name
+		return @name
+	end
+
+	def size
+		return @list.size
+	end
+
+	#if completed, include items that are done
+	def priority_list(completed = false)
```

Comment:
I like that you used methods on the list class to sort the items. Definitely the proper place for this logic.